assets/js/modules/adsense/index.js

```diff
@@ -73,6 +73,7 @@ domReady( () => {
 	Modules.registerModule(
 		'adsense',
 		{
+			name: 'AdSense',
```

Comment:
Why add the module `name` here? This is part of the server-side module information and shouldn't be registered in JS. If you added this due to tests failing, let's find another way to make these work. The `provideModules` utility function (via the `receiveGetModules` datastore action) should set this information up correctly.

Reply:
For whatever reason when I was working on that, the name had the same value as the slug. I removed the name from here and looks to be fine now. Thanks for letting me know.